app/init/credentials.ts

```diff
@@ -4,16 +4,56 @@
 import {Platform} from 'react-native';
 import AsyncStorage from '@react-native-community/async-storage';
```

Comment:
When sorting imports, the `@react-native-community/async-storage` should be treated as the character `a` and sorted amongst the first modules.